ibm_platform_services/context_based_restrictions_v1.py

```diff
@@ -59,7 +59,9 @@ def new_instance(
                specified parameters and external configuration.
         """
         authenticator = get_authenticator_from_environment(service_name)
-        service = cls(authenticator)
+        service = cls(
+            authenticator
+            )
```

Comment:
This seems like odd formatting, did this pass the PEP8 linting?

Reply:
Linting has been done